apps/dashboard/src/components/workflow-editor/steps/edit-step-template.tsx

```diff
@@ -0,0 +1,10 @@
+import { ConfigureStepTemplate } from '@/components/workflow-editor/steps/configure-step-template';
```

Comment:
Are these edit components necessary? I feel that we can move the wrapping into the  Configure* components and remove two extra imports.

Reply:
The component is necessary because it wraps with the provider. The file isn't that much but I liked it separated as it's the entry point in main.tsx